pkgs/tools/typesetting/tex/pplatex/default.nix

```
@@ -0,0 +1,49 @@
+{
+	pkgs,
+	fetchFromGitHub,
+	stdenv,
+	cmake,
+	pkgconfig,
+	pcre,
+	...
+} :
+	stdenv.mkDerivation {
+		pname = "fop";
+		version = "1.0-rc2";
+
+		src = fetchFromGitHub {
+			owner = "stefanhepp";
+			repo = "pplatex";
+			rev = "25bf7e121178f1fee2452b4fa9961248a045a387";
```

```suggestion
			rev = "pplatex-${version}";
```

---

But actually, why not use the latest commit?
```suggestion
			rev = "5cec891ad6aec0115081cdd114ae1cc4f1ed7c06";
```